### PR TITLE
[backport] Use collections.abc to avoid DeprecationWarning in Python 3.7

### PR DESCRIPTION
**Pull request by beam2d**
Backport of #5172

### CODE REVIEW

**Comment by beam2d** [comment]
[automatic post] Jenkins, test this please.

File: chainer/functions/normalization/batch_normalization.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 from chainer import function_node
 from chainer.utils import argument
 from chainer.utils import type_check
+from chainer.utils import collections_abc
```
Comment (toslunar) [inline chainer/functions/normalization/batch_normalization.py]: This import seems unused.

Comment (beam2d) [reply chainer/functions/normalization/batch_normalization.py]: Fixed.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
@toslunar Could you check the import order error in travis?

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
LGTM!

**Pull request merged**